### ISSUE TITLE
speccy lint fails: UnhandledPromiseRejectionWarning: ReferenceError: message is not defined

### ISSUE THREAD

**Issue by techieshark** [wework/speccy#81]
## Detailed description

Seems to be a bug in the `speccy` code - `message` is not always defined, so can cause crash when codes accesses it.

## Context

Would be helpful if speccy provided useful error message rather than crashing.

```
$ speccy lint --skip openapi-tags --skip operation-tags openapi.yaml
Specification schema is invalid.
(node:12112) UnhandledPromiseRejectionWarning: ReferenceError: message is not defined
    at err.errors.map.error (myproject/node_modules/speccy/lint.js:50:31)
    at Array.map (<anonymous>)
    at readableJsonSchemaMessages (myproject/node_modules/speccy/lint.js:39:23)
    at formatSchemaError (myproject/node_modules/speccy/lint.js:30:32)
    at validator.validate (myproject/node_modules/speccy/lint.js:97:28)
    at myproject/node_modules/speccy/lib/validator.js:1274:9
    at <anonymous>
(node:12112) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
(node:12112) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
✨  Done in 0.93s.
```
Appears to be thrown by this line:
https://github.com/wework/speccy/blob/63cb0cee63893b0039bf4eb8335abe29aa4041fb/lint.js#L50

## Possible implementation

Not sure yet.

## Your environment

Include as many relevant details about the environment you experienced the bug in and how to reproduce it.

* Node Version: 8.11.1
* Operating system and version (e.g. Ubuntu 16.04, Windows 7): OS X

**Comment by philsturgeon** [comment]
Already fixed in https://github.com/wework/speccy/commit/f939a9f6395077a1422233d02003e0314534422a

I’ll tag up 0.7.2 tomorrow but I’d like #80 in there too.

**Comment by philsturgeon** [comment]
This fix is now released as part of v0.7.2.

**Comment by techieshark** [comment]
Awesome, thanks @philsturgeon!
